### ISSUE TITLE
onos-ric crash when accessing atomix store

### ISSUE THREAD

**Issue by SeanCondon** [onosproject/onos-ric#105]
Not sure why, but I get the following from `onos-ric`
```
I0228 18:23:48.072652       1 manager.go:189] RadioMeasReport plmnid:001001 ecid:0000005 crnti:0125 cqis:11(ecid:0000002),4(ecid:0000001),4(ecid:0000005)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xc5e816]

goroutine 694 [running]:
github.com/onosproject/onos-ric/pkg/store/telemetry.(*atomixStore).Get(0xc00027c7d0, 0x12, 0xc0004a5d34, 0x6, 0xc0004a5d40, 0x7, 0xc0004a5d30, 0x4, 0x0, 0x0, ...)
	/go/src/github.com/onosproject/onos-ric/pkg/store/telemetry/store.go:144 +0x2b6
github.com/onosproject/onos-ric/pkg/manager.(*Manager).DeleteTelemetry(0x1603fa0, 0xc0004a5d34, 0x6, 0xc0004a5d40, 0x7, 0xc0004a5d30, 0x4, 0xe38ea0, 0xc0005127c0)
	/go/src/github.com/onosproject/onos-ric/pkg/manager/manager.go:214 +0xee
github.com/onosproject/onos-ric/pkg/northbound/c1.Server.TriggerHandOver(0xf860c0, 0xc0004a7b60, 0xc0005127c0, 0xdac300, 0xc000484d00, 0xf87c40)
	/go/src/github.com/onosproject/onos-ric/pkg/northbound/c1/c1.go:226 +0x21d
github.com/onosproject/onos-ric/api/nb._C1InterfaceService_TriggerHandOver_Handler(0xdac300, 0x1620b88, 0xf860c0, 0xc0004a7b60, 0xc000484d80, 0x0, 0xf860c0, 0xc0004a7b60, 0xc00028e570, 0x2c)
	/go/src/github.com/onosproject/onos-ric/api/nb/c1-interface.pb.go:2718 +0x217
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000171680, 0xf8d3e0, 0xc000171e00, 0xc00038d400, 0xc00017db00, 0x15ecea0, 0x0, 0x0, 0x0)
	/go/src/github.com/onosproject/onos-ric/vendor/google.golang.org/grpc/server.go:1024 +0x4f4
google.golang.org/grpc.(*Server).handleStream(0xc000171680, 0xf8d3e0, 0xc000171e00, 0xc00038d400, 0x0)
	/go/src/github.com/onosproject/onos-ric/vendor/google.golang.org/grpc/server.go:1313 +0xd97
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000270a10, 0xc000171680, 0xf8d3e0, 0xc000171e00, 0xc00038d400)
	/go/src/github.com/onosproject/onos-ric/vendor/google.golang.org/grpc/server.go:722 +0xbb
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/onosproject/onos-ric/vendor/google.golang.org/grpc/server.go:720 +0xa1
```

**Comment by adibrastegarnia** [comment]
@SeanCondon 
It looks like tries to process a value from the store that doesn't exist in the store.

**Comment by adibrastegarnia** [comment]
@SeanCondon 
How we can reproduce it?

**Comment by SeanCondon** [comment]
I ran the `ran-simulator`, `onos-ric` and `onos-ric-ho` and `sd-ran-gui` together.

I followed the logs of `ran-simulator` and `onos-ric`. You will have to run the `ran-simulator` with the 
https://github.com/onosproject/ran-simulator/pull/56 patch

I opened the GUI and increased the number of UEs first to 90 then to 300.

I brought it back down to 90 and `onos-ric` crashed like this. It doesn't happen every time.
